instrumentation/delayed_job/lib/opentelemetry/instrumentation/delayed_job/instrumentation.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module DelayedJob
+      # Instrumentation class that detects and installs the DelayedJob instrumentation
+      class Instrumentation < OpenTelemetry::Instrumentation::Base
+        MINIMUM_VERSION = Gem::Version.new('4.1')
+
+        install do |_config|
+          require_dependencies
+          register_tracer_middleware
+        end
+
+        present do
+          !defined?(::Delayed).nil? && gem_version >= MINIMUM_VERSION
```

Comment:
Curious why you're doing the explicit `nil` check here, we're doing `defined?(::Library)` everywhere else.  Are we missing an edge case?

Reply:
I want this method to return exactly true or false (rather than nil).
* `defined?(::Delayed)` can return nil, so can't use it
* Rubocop doesn't like `!!defined?(::Delayed)`
* Therefore only remaining option is `!defined?(::Delayed).nil?`